assert/assertions_test.go

```diff
@@ -182,49 +177,37 @@ func TestIsType(t *testing.T) {
 
 }
 
-type myType string
```

Comment:
I don't like to pollute the top namespace if not necessary

Reply:
Agreed, good one.